frontend/src/metabase/query_builder/components/NewDatasetModal/NewDatasetModal.jsx

```diff
@@ -30,33 +33,37 @@ function NewDatasetModal({ turnQuestionIntoDataset, onClose }) {
 
   return (
     <ModalContent
-      title={t`Create datasets to make it easier for everyone to explore.`}
       footer={[
-        <Button key="cancel" onClick={onClose}>{t`Cancel`}</Button>,
+        <Link
+          className="text-brand"
+          key="cancel"
+          onClick={onClose}
+        >{t`Cancel`}</Link>,
         <Button
           key="action"
           primary
           onClick={onConfirm}
         >{t`Turn this into a dataset`}</Button>,
       ]}
     >
-      <DatasetFeaturesContainer>
-        <DatasetFeatureOverview icon="dataset">
-          {jt`You’ll see them in the ${(
-            <strong>{t`Datasets section`}</strong>
-          )} when creating a new question.`}
-        </DatasetFeatureOverview>
-        <DatasetFeatureOverview icon="folder">
-          {jt`Easily ${(
-            <strong>{t`open a dataset from its collection`}</strong>
-          )} or via Search to start a new question.`}
-        </DatasetFeatureOverview>
-        <DatasetFeatureOverview icon="label">
-          {jt`You can ${(
-            <strong>{t`customize a dataset’s metadata`}</strong>
-          )} to make it even easier to explore the data.`}
-        </DatasetFeatureOverview>
-      </DatasetFeaturesContainer>
+      <FeatureOverviewContainer>
+        <DatasetImg src="app/img/dataset-illustration.svg" />
+        <DatasetTitle>{t`Datasets`}</DatasetTitle>
+        <ul>
+          <DatasetValueProp>
+            Let you update column descriptions and customize metadata to create
+            great starting points for exploration.
+          </DatasetValueProp>
+          <DatasetValueProp>
+            Show up higher in search results and get highlighted when other
+            users start new questions to promote reuse.
+          </DatasetValueProp>
+          <DatasetValueProp>
+            Live in collections to keep them separate from messy database
+            schemas.
+          </DatasetValueProp>
```

Comment:
We need to wrap the copy with `t`, so they can be translated

Reply:
Whoops! Totally forgot, thanks!